.github/workflows/mdbook.yml

```diff
@@ -1,8 +1,6 @@
-name: github pages
+name: test mdbook
```

Comment:
I assume this action is for testing whether the book comiles?

Reply:
yes